openstack/containerinfra/v1/services/requests.go

```diff
@@ -0,0 +1,13 @@
+package services
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+func List(client *gophercloud.ServiceClient) pagination.Pager {
```

Comment:
Per the code convention [1], could you make sure to document all public functions, types and fields?

[1] https://github.com/gophercloud/gophercloud/blob/master/docs/contributor-tutorial/step-05-pull-requests.md#documentation